TEMPLATE.md

```diff
@@ -0,0 +1,652 @@
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+
+    }
+}
+```
+
+### Doctrine ###
+
+#### Classes ####
+
+`sfentityclass`
+
+``` php
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="")
+ * @ORM\Table(name="")
+ */
+class Entity
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    protected $id;
+}
+```
+
+`sfdocumentclass`
+
+``` php
+use Doctrine\ODM\MongoDB\Mapping\Annotations as MongoDB;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @MongoDB\Document(collection="", repositoryClass="")
+ */
+class Document
+{
+    /**
+     * @MongoDB\Id()
+     */
+    protected $id;
+}
+```
+
+`sfrepository`
+
+``` php
+use Doctrine\ORM\EntityRepository;
+
+class EntityNameRepository extends EntityRepository
+{
+}
+```
+
+`sfgetset`
+Just type your variable name (firstName for instance), and the snippets will name the functions automatically (getFirstName).
+
+``` php
+/**
+* Get 
+* @return  
+*/
+public function get()
+{
+    return $this->;
+}
+
+/**
+* Set 
+* @return  
+*/
+public function set($)
+{
+    $this-> = $;
+    return $this;
+}
+```
+
+#### Annotations ####
+
+After triggering the snippets, just type your var name and it will automatically set the name in the annotation (If you type 'firstName', your variable will be named 'firstName' and in the annotation the name will be 'first_name').
+
+`sfentity`
+
+``` php
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="name")
+ */
+```
+
+`sfidcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="id", type="integer", nullable=false)
+ * @ORM\Id()
+ * @ORM\GeneratedValue(strategy="IDENTITY")
+ */
+ private $id;
+```
+
+`sfstringcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="string", length=255)
+ */
+ private $;
+```
+
+`sfdecimalcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="decimal", scale=2)
+ */
+ private $;
+```
+
+`sftextcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="text")
+ */
+ private $;
+```
+
+`sfintegercolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="integer")
+ */
+ private $;
+```
+
+`sfbooleancolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="boolean")
+ */
+ private $;
+```
+
+`sfsmallintcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="smallint")
+ */
+ private $;
+```
+
+`sfbigintcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="bigint")
+ */
+ private $;
+```
+
+`sfdatetimecolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="datetime")
+ */
+ private $;
+```
+
+`sfdatecolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="date")
+ */
+ private $;
+```
+
+`sftimecolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="time")
+ */
+ private $;
+```
+
+`sffloatcolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="float")
+ */
+ private $;
+```
+
+`sfarraycolumn`
+
+``` php
+/**
+ * @ORM\Column(name="", type="array")
+ */
+ private $;
```

Comment:
I'm still undecided - that's part of the reason I made search the new README is perfect - so we can stare at all of them together and see how they feel